tests/components/laundrify/test_sensor.py

```diff
@@ -0,0 +1,106 @@
+"""Test the laundrify sensor platform."""
+
+from datetime import timedelta
+import json
+import logging
+from unittest.mock import AsyncMock, patch
+
+from laundrify_aio import LaundrifyDevice
+import pytest
+
+from homeassistant.components.laundrify.const import DOMAIN, MANUFACTURER, MODELS
+from homeassistant.components.laundrify.sensor import LaundrifyPowerSensor
+from homeassistant.components.sensor import SensorDeviceClass
+from homeassistant.const import (
+    ATTR_DEVICE_CLASS,
+    ATTR_UNIT_OF_MEASUREMENT,
+    STATE_UNKNOWN,
+    UnitOfPower,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr
+from homeassistant.util import slugify
+from homeassistant.util.dt import utcnow
+
+from . import setup_config_entry
+
+from tests.common import async_fire_time_changed, load_fixture
+
+# Add logging to trace the flow
+logging.basicConfig(level=logging.DEBUG)
+
+
+@pytest.fixture(name="laundrify_sensor")
+def laundrify_sensor_fixture():
+    """Return a default Laundrify power sensor mock."""
+    # Load test data from machines.json
+    machine_data = json.loads(load_fixture("laundrify/machines.json"))[0]
+
+    mock_device = AsyncMock(spec=LaundrifyDevice)
+    mock_device.id = machine_data["id"]
+    mock_device.manufacturer = MANUFACTURER
+    mock_device.model = machine_data["model"]
+    mock_device.name = machine_data["name"]
+    mock_device.firmwareVersion = machine_data["firmwareVersion"]
+    return LaundrifyPowerSensor(mock_device)
+
+
+async def test_laundrify_sensor_init(
+    laundrify_sensor, hass: HomeAssistant, device_registry: dr.DeviceRegistry
```

Comment:
The `laundrify_sensor` fixture could be typed to make it complete

Reply:
Addressed in https://github.com/home-assistant/core/pull/121378/commits/6d61111b4a4f529eb108a37c18f2ebed227b1e11